aoc/src/main/scala/AoC2015Day03.scala

```diff
@@ -0,0 +1,42 @@
+object AoC2015Day03 extends App:
+
+    import scala.io.Source
+
+    val directions: String =
+        Source.fromResource("2015/InputDay03.txt")
+        .getLines
+        .toList.head
+
+    case class House(x: Int, y: Int)
+
+    // Part 1
+
+    def findVisitedHouses(curHouse: House, visitedHouses: Set[House], directionsStr: String): Set[House] =
+        if (directionsStr.isEmpty) visitedHouses.incl(curHouse)
```

Comment:
nice - for alternatives - next time try a pattern match on a string as a list instead of an if statement, eg.

```
list match
  case head :: tail => // binds head and tail
  case Nil          => // empty list
```